package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@daily-co/daily-js": "^0.41.0",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
-    "@testing-library/user-event": "^7.2.1",
-    "react": "^16.13.0",
-    "react-dom": "^16.13.0",
-    "react-scripts": "3.4.0",
-    "serve": "^11.3.0"
+    "@daily-co/daily-js": "^0.43.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
+    "react": "^18.2.0",
```

Comment:
I'm surprised this works 🤔 is there no warning in the console? When upgrading to 18 ReactDOM.render is no longer supported, so we might need to [adjust this file](https://github.com/daily-demos/call-object-react/blob/main/src/index.js):


```
import React from 'react';
import ReactDOM from 'react-dom';
import './index.css';
import App from './components/App/App';
import BrowserUnsupported from './components/BrowserUnsupported/BrowserUnsupported';
import DailyIframe from '@daily-co/daily-js';

ReactDOM.render(
  DailyIframe.supportedBrowser().supported ? <App /> : <BrowserUnsupported />,
  document.getElementById('root')
);
```

to:

```
import { createRoot } from 'react-dom/client';
import DailyIframe from '@daily-co/daily-js';
import './index.css';
import App from './components/App/App';
import BrowserUnsupported from './components/BrowserUnsupported/BrowserUnsupported';

const container = document.getElementById('app');
const root = createRoot(container); // createRoot(container!) if you use TypeScript
root.render(DailyIframe.supportedBrowser().supported ? <App /> : <BrowserUnsupported />);
```

(not tested, could be buggy)

Reply:
If it still works, I think that's because React is treating it as if React 17 was installed: https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis

Reply:
Ahh yes you are right, there is a warning that I missed:

```
Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot
printWarning @ react-dom.development.js:86
logUtils.js:2 [daily.co event] participant-updated
logUtils.js:2 [daily.co event] participant-updated
logUtils.js:2 [daily.co event] joined-meeting
```

Will try your code above now